libfastpm/cosmology.c

```diff
@@ -376,18 +400,20 @@ double DGrowthFactorDa(double a, FastPMCosmology * c) {
     return F1 / a / d0;
 }
 
-double D2GrowthFactorDa2(double a, FastPMCosmology * c) {
+double D2GrowthFactorDa2(FastPMGrowthInfo * growth_info) {
+    /* d^2 D1 / da^2 */
+    double a = growth_info->a;
+    FastPMCosmology * c = growth_info->c;
+
     double E = HubbleEa(a, c);
     double dEda = DHubbleEaDa(a, c);
-    
-    double d1 = growth(a, c);
-    double F1 = DgrowthDlna(a, c);
-    double d0 = growth(1., c);  //0 for today
+    double D1 = growth_info->D1;
+    double f1 = growth_info->f1;
     
     double ans = 0.;
-    ans -= (3. + a / E * dEda) * F1;
-    ans += 1.5 * Omega_cdm_a(a, c) * d1;
-    ans /= d0 * a*a;
+    ans -= (3. + a / E * dEda) * f1 * D1;
+    ans += 1.5 * Omega_m(a, c) * D1;
```

Comment:
Note this is now Omega_m

Reply:
Yes it looks like Omega_m makes more sense -- what was the argument for using cdm alone previously?
How big of an impact this is ? A small correction on second order?

Reply:
Changing this alone would cause quite a big difference (multiple %), but then if you change kick->dda to Omega_m as well the difference is very small indeed, because dda ~ 1/g_f ~ 1/(d^2D/da^2).

 I could study this further if you like?

Reply:
Also, there was no argument for using Omega_cdm before, it just carried over from the old code.